test/e2e/docker/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apt-get -qq update -y && apt-get -qq upgrade -y >/dev/null
 RUN apt-get -qq install -y iputils-ping iproute2 >/dev/null
 
 # Set up build directory /src/cometbft
-ENV COMETBFT_BUILD_OPTIONS badgerdb,rocksdb,clock_skew,bls12381
+ENV COMETBFT_BUILD_OPTIONS=badgerdb,rocksdb,clock_skew,bls12381
```

Comment:
Thanks for spotting. Can you also fix `GORACE` and `CMTHOME`? (I see they also have spaces)